README.md

```diff
@@ -839,6 +839,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Ghada A.Saleh](https://github.com/ghadasaleh3)
   - [Gianluca Galota](https://github.com/ThorOnTheRocks)
   - [Gideon Idoko](https://github.com/IamGideonIdoko)
+  - [Gideon Chidi](https://github.com/gideon-del)
```

Comment:
Your name must be above Gideon Idoko.
Let us know if you need any help with that. 🙂